app/config/config_test.yml

```diff
@@ -35,6 +35,10 @@ services:
         arguments:
             - '%env(SSL_PRIVATE_KEY)%'
 
+    app.committee.manager:
```

Comment:
why is it public? :D

Reply:
because tests.
https://github.com/EnMarche/en-marche.fr/blob/master/tests/Controller/EnMarche/CommitteeManagerControllerTest.php#L601